frontend/src/Form/beta/ModelCardForm.tsx

```diff
@@ -41,6 +54,13 @@ export default function ModelCardForm({
     return <></>
   }
 
+  function listButtonOnClick(index: number, formPageKey: string) {
```

Comment:
Again, shout if you'd rather not follow this convention, but consider renaming to `handleListItemClick`